com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfRobotExtensions.cs

```diff
@@ -0,0 +1,363 @@
+﻿/*
+© Siemens AG, 2018
+Author: Suzannah Smith (suzannah.smith@siemens.com)
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+<http://www.apache.org/licenses/LICENSE-2.0>.
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System;
+using System.IO;
+using System.Linq;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+using UnityEngine;
+
+namespace RosSharp.Urdf
+{
+    public static class UrdfRobotExtensions
+    {
+        static string tagName = "robot";
+        static string collisionObjectName = "Collisions";
+        public static ImportSettings importsettings;
+
+        public static void Create()
+        {
+            CreateTag();
+            GameObject robotGameObject = new GameObject("Robot");
```

Comment:
nit: why do we want to arbitrarily set the name of the gameObject?

Reply:
making it just a bit easier to find in the inspector.